### PR TITLE
Verify jekyll output

### PR DESCRIPTION
**Pull request by NullableInt**

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+end
+
+task :clean do
+  puts 'Cleaning up _site...'.bold
+  Jekyll::Commands::Clean.process({})
+end
+
+# Test generated output has valid HTML and links.
+task :test do
+  Jekyll::Commands::Build.process(profile: true)
```
Comment (asbjornu) [inline Rakefile:25]: Thinking twice about this: Isn't it possible to declare that one Rake task depends on another, somehow? `:test` depends on `:build` would be the proper way to do this, I think.

Comment (NullableInt) [reply Rakefile:25]: Can do!

**Comment by NullableInt** [comment]
Have fun!

File: payments/vipps/other-features.md [hunk 2]
```diff
@@ -240,24 +240,25 @@ All Vipps error types will have the following URI in front of type:
...
+| *ACQUIRER_INSUFFICIENT_FUNDS* | 403    |      |
+| *ACQUIRER_INVALID_AMOUNT*     | 403    |      |
+| *ACQUIRER_POSSIBLE_FRAUD*     | 403    |      |
+| *FRAUD_DETECTED*              | 403    |      |
+| *BAD_REQUEST*                 | 500    |      |
+| *INTERNAL_SERVER_ERROR*       | 500    |      |
+| *BAD_GATEWAY*                 | 502    |      |
+| *ACQUIRER_GATEWAY_ERROR*      | 502    |      |
+| *ACQUIRER_GATEWAY_TIMEOUT*    | 504    |      |
+| *UNKNOWN_ERROR*               | 500    |      |
```
Comment (asbjornu) [inline payments/vipps/other-features.md:261]:
```suggestion
| Type                            | Status |
| :------------------------------ | :----- |
| *`CARD_BLACKLISTED`*            | `400`  |
| *`PAYMENT_TOKEN_ERROR`*         | `403`  |
| *`CARD_DECLINED`*               | `403`  |
| *`ACQUIRER_ERROR`*              | `403`  |
| *`ACQUIRER_CARD_BLACKLISTED`*   | `403`  |
| *`ACQUIRER_CARD_EXPIRED`*       | `403`  |
| *`ACQUIRER_CARD_STOLEN`*        | `403`  |
| *`ACQUIRER_INSUFFICIENT_FUNDS`* | `403`  |
| *`ACQUIRER_INVALID_AMOUNT`*     | `403`  |
| *`ACQUIRER_POSSIBLE_FRAUD`*     | `403`  |
| *`FRAUD_DETECTED`*              | `403`  |
| *`BAD_REQUEST`*                 | `500`  |
| *`INTERNAL_SERVER_ERROR`*       | `500`  |
| *`BAD_GATEWAY`*                 | `502`  |
| *`ACQUIRER_GATEWAY_ERROR`*      | `502`  |
| *`ACQUIRER_GATEWAY_TIMEOUT`*    | `504`  |
| *`UNKNOWN_ERROR`*               | `500`  |
```

File: .github/workflows/verify_build_output.yml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    name: Jekyll output
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Docker compose up with rake
```
[the remark this answers is not in the document]
Comment (NullableInt) [reply .github/workflows/verify_build_output.yml:11]: Check `.docker/docker-compose.test.yml` it's in there

**Pull request merged**